src/cryptography/hazmat/primitives/asymmetric/rsa.py

```diff
@@ -249,8 +255,6 @@ def rsa_recover_prime_factors(n: int, e: int, d: int) -> tuple[int, int]:
                 spotted = True
                 break
             k *= 2
-        # This value was not any good... let's try another!
-        a += 2
```

Comment:
When we removed this we didn't update the corresponding `_MAX_RECOVERY_ATTEMPTS` and associated comment, which talks about this double increment.

Reply:
Also fixed.